src/views/tickets/view-link.php

```diff
@@ -55,13 +56,13 @@
 	return false;
 }
 
-$link = $view->get_tickets_page_url( $event_id, $is_event_page );
+$link = $view->get_tickets_page_url( $post_id, $is_event_page );
 
 // Translators: 1: number of RSVPs and/or Tickets with accompanying ticket type text, 2: post type label.
 $message = esc_html( sprintf( __( 'You have %1s for this %2s.', 'event-tickets' ), implode( _x( ' and ', 'separator if there are both RSVPs and Tickets', 'event-tickets' ), $counters ), $post_type_singular ) );
 ?>
 
 <div class="tribe-link-view-attendee">
 	<?php echo $message ?>
-	<a href="<?php echo esc_url( $link ) ?>"><?php echo sprintf( esc_html__( 'View your %s', 'event-tickets' ), $view->get_description_rsvp_ticket( $event_id, $user_id ) ) ?></a>
-</div>
\ No newline at end of file
```

Comment:
* :no_entry_sign: Inline PHP statement must end with a semicolon
* :no_entry_sign: Inline PHP statement must end with a semicolon
* :no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$view'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$view'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$user_id'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$user_id'.

<sub>hash: 1990b441e8d126d73a3b47cf095c345e</sub>